distributions/openhab/src/main/resources/runtime/etc/jetty.xml

```diff
@@ -79,10 +79,10 @@
 		</Arg>
 	</Call>
 
-	<New id="httpConfig" class="org.eclipse.jetty.server.HttpConfiguration">
```

Comment:
No need to change the name - in my example I only changed it because I added a second server configuration to keep http/1.1 support.

Reply:
Reverted :+1: